ios/Surveys/Sources/Presentation/Modules/Home/HomeHeaderView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  HomeHeaderView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Kingfisher
+import SwiftUI
+
+struct HomeHeaderView: View {
+
+    private let title: String
+    private let imageURLString: String
+
+    var body: some View {
+        HStack(alignment: .center) {
+            VStack(
+                alignment: .leading,
+                spacing: 10.0
+            ) {
+                Text(title)
+                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 13.0))
+                    .foregroundColor(.white)
+                Text(Localize.homeHeaderToday())
+                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 34.0))
+                    .foregroundColor(.white)
+            }
+            Spacer()
+            KFImage(URL(string: imageURLString))
```

Comment:
Just a suggestion, I think we should create a custom view named `ImageView` or something like that. The purpose is to wrap the lib's stuff with URL and placeholder as parameters. What do you think?

Reply:
Noted! I will improve this on incoming pull requests